packages/component/src/Activity/StackedLayout.js

```diff
@@ -117,7 +119,11 @@ const StackedLayout = ({ activity, children, timestampClassName }) => {
   const botAriaLabel = useLocalize('Bot said something', initials, plainText);
   const userAriaLabel = useLocalize('User said something', initials, plainText);
 
-  const ariaLabel = fromUser ? userAriaLabel : botAriaLabel;
+  const sentAtTimestamp = useLocalize('SentAt') + CalculateAbsoluteTime(timestamp);
+
+  const someoneSaidString = (fromUser ? userAriaLabel : botAriaLabel).trim();
```

Comment:
Not a huge fan of that variable name

Reply:
Hmm I suppose, but we have variables in other places saying "BotSaidx" and "UserSaidx", so it's not like there isn't precedent. 

Got other suggestions?

Reply:
Lol no